README.md

```diff
@@ -47,23 +57,17 @@ const zaproxy = new ZapClient(zapOptions);
 
 ### Use the Node API:
 
-Callbacks:
-
-```js
-zaproxy.spider.scanAsUser(contextId, userId, sutBaseUrl, maxChildren, recurse, subtreeonly, (err, resp) => {
-  if (err) // Handle the error.
-  if (resp) // Handle the response.
-});
-```
-
-Promises:
-
 ```js
-await zaproxy.spider.scanAsUser(contextId, userId, sutBaseUrl, maxChildren)
-  .then(
-    resp => console.log(JSON.stringify(resp)),
-    err => `Error occurred while attempting to scan as user. Error was: ${err.message}`
-  );
+let params = {
+  contextid,
+  userid,
+  sutbaseurl,
```

Comment:
The parameter needs to be mapped.

Reply:
Can you explain `mapped` a bit more?

Reply:
Done

Reply:
For the others they didn't need to be changed (though it will work fine as is).